### PR TITLE
add rule for mobile01.com

### PR DESCRIPTION
**Pull request by PeterDaveHello**

### CODE REVIEW

**Comment by reedy** [comment]
Visiting https://www.mobile01.com seems to get redirected back to http://www.mobile01.com

**Comment by PeterDaveHello** [comment]
hmmmm ... I should setup rule for login/register page only.

**Comment by donced** [comment]
@ https://github.com/EFForg/https-everywhere/pull/1632#issuecomment-100764397
what's the point, it's still trivial to intercept the cookie even with such rule.

**Comment by semenko** [comment]
Closed per #2237

**Comment by PeterDaveHello** [comment]
hmmm ... what about let me update this PR?

**Comment by semenko** [comment]
Awesome! You're definitely welcome to modify / update this PR! :)

It looks like this rule was also in PR #2237, which includes some login fixes you talked about above.

**Comment by PeterDaveHello** [comment]
But I can not reopen a closed PR, can you open it?

**Comment by semenko** [comment]
Sure -- though you should be able to push changes to your branch without this PR being open (and you can always open a new PR.)

**Comment by PeterDaveHello** [comment]
Updated!

File: src/chrome/content/rules/mobile01.com.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<ruleset name="mobile01.com">
+	<target host="mobile01.com" />
+	<target host="www.mobile01.com" />
+
+	<test url="http://www.mobile01.com/login.php">
```
Comment (donced) [inline src/chrome/content/rules/mobile01.com.xml]: missing slash, `<test url="http://www.mobile01.com/login.php" />`

**Comment by donced** [comment]
PR https://github.com/EFForg/https-everywhere/pull/2237 already addressed https://github.com/EFForg/https-everywhere/pull/1632#issuecomment-100764397. This PR currently doesn't seem to make any improvement on the newly [merged rule](https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/Mobile01.com.xml) for mobile01.com. The reason I made another PR because it had been more than 2 months since https://github.com/EFForg/https-everywhere/pull/1632#issuecomment-100764397, so I assumed this PR has been abandoned when I create https://github.com/EFForg/https-everywhere/pull/2237.

**Comment by PeterDaveHello** [comment]
Fine, in fact #2237 seems to have some unrelated changes.

**Pull request closed without merging**